### DISCUSSION TITLE
Dealing with /config/hahm/unignore file during migration

### DISCUSSION THREAD

**Discussion by sotatech** [SukramJ/aiohomematic#379] [General]
Congratulation on the release of version 1 of this integration! 
In the migration instructions you mention deleting the /config/hahm folder after removing the old version. I have an _unignore_ file in this folder to record some additional parameters for my Heating Groups.

Where should this file be saved after the migration? Also, it might be worth telling users to check for this file before deleting the folder.

**Comment by SukramJ** [comment]
Thanks.
Move it to /config/homematicip_local

**Reply by sotatech** [reply]
Worked perfectly, thanks.
